mariadb/manifest.jsonnet

```diff
@@ -51,25 +51,96 @@ kpm.package({
   resources: [
     // Config maps.
     {
-      file: "configmaps/my.cnf.yaml.j2",
-      template: (importstr "templates/configmaps/my.cnf.yaml.j2"),
-      name: "mariadb-mycnf",
+      file: "configmaps/seed.sh.yaml.j2",
+      template: (importstr "templates/configmaps/seed.sh.yaml.j2"),
+      name: "seedsh",
       type: "configmap",
     },
 
     {
       file: "configmaps/start.sh.yaml.j2",
       template: (importstr "templates/configmaps/start.sh.yaml.j2"),
-      name: "mariadb-startsh",
+      name: "startsh",
       type: "configmap",
     },
 
-    // Deployments.
     {
-      file: "deployment.yaml.j2",
-      template: (importstr "templates/deployment.yaml.j2"),
-      name: "mariadb",
-      type: "deployment",
+      file: "configmaps/peer-finder.py.yaml.j2",
+      template: (importstr "templates/configmaps/peer-finder.py.yaml.j2"),
+      name: "mariadb-peer-finder",
+      type: "configmap",
+    },
+
+    {
+      file: "configmaps/charsets.cnf.yaml.j2",
+      template: (importstr "templates/configmaps/charsets.cnf.yaml.j2"),
+      name: "mariadb-charsets",
+      type: "configmap",
+    },
+
+    {
+      file: "configmaps/engine.cnf.yaml.j2",
+      template: (importstr "templates/configmaps/engine.cnf.yaml.j2"),
+      name: "mariadb-engine",
+      type: "configmap",
+    },
+
+    {
+      file: "configmaps/galera-my.yaml.j2",
+      template: (importstr "templates/configmaps/galera-my.cnf.yaml.j2"),
+      name: "mariadb-mycnf",
+      type: "configmap",
+    },
+
+    {
+      file: "configmaps/log.cnf.yaml.j2",
+      template: (importstr "templates/configmaps/log.cnf.yaml.j2"),
+      name: "mariadb-log",
+      type: "configmap",
+    },
+
+    {
+      file: "configmaps/networking.cnf.yaml.j2",
+      template: (importstr "templates/configmaps/networking.cnf.yaml.j2"),
+      name: "mariadb-networking",
+      type: "configmap",
+    },
+
+    {
+      file: "configmaps/pid.cnf.yaml.j2",
+      template: (importstr "templates/configmaps/pid.cnf.yaml.j2"),
+      name: "mariadb-pid",
+      type: "configmap",
+    },
+
+    {
+      file: "configmaps/tuning.cnf.yaml.j2",
+      template: (importstr "templates/configmaps/tuning.cnf.yaml.j2"),
+      name: "mariadb-tuning",
+      type: "configmap",
+    },
+
+    {
+      file: "configmaps/wsrep.cnf.yaml.j2",
+      template: (importstr "templates/configmaps/wsrep.cnf.yaml.j2"),
+      name: "mariadb-wsrep",
+      type: "configmap",
+    },
+
+    // Jobs.
+    {
+      file: "init.yaml.j2",
+      template: (importstr "templates/init.yaml.j2"),
+      name: "mariadb-init",
+      type: "job",
+    },
+
+    // DaemonSets.
+    {
+      file: "daemonset.yaml.j2",
+      template: (importstr "templates/daemonset.yaml.j2"),
+      name: "mariadb-ds",
+      type: "daemonset",
```

Comment:
Agreed with the team (@ss7pro) to leave it as a DaemonSet for now. It'd require some additional work to setup Kubernetes 1.4 (for MariaDB running as a Deployment it's necessary to have anti-affinity feature which prevents the PODs to be scheduled on the same node). Therefore I'll create a new Jira item to replace current DaemonSet implementation with Deployment - it'll depend on Kubernetes 1.4 setup item.